src/components/BooksForm.js

```diff
@@ -25,7 +25,7 @@ const BooksForm = () => {
   };
```

Comment:
You can refactor handleChange to look like so:

``` 
handleChange({ target: { name, value } }) {
  this.setState({
    [name]: value,
  })
}
  ```

Reply:
Destructed event